package.json

```diff
@@ -7,8 +7,9 @@
   "exports": "./dist/index.modern.js",
   "module": "dist/index.module.js",
   "unpkg": "dist/index.umd.js",
+  "types": "dist/index.d.ts",
```

Comment:
why not just `index.d.ts`? why copy to dist? npm publish will still have the type defs at the root folder and do the lookup right?

Reply:
You're right! My bad - updated